ios/FluentUI/Core/FluentUIFramework.swift

```diff
@@ -56,11 +34,6 @@ public class FluentUIFramework: NSObject {
     @available(*, deprecated, message: "Non-fluent icons no longer supported. Setting this var no longer has any effect and it will be removed in a future update.")
     @objc public static var usesFluentIcons: Bool = true
 
-    @available(*, deprecated, renamed: "initializeAppearance(with:whenContainedInInstancesOf:)")
```

Comment:
are we ready for deprecating in this pr?

Reply:
I can add it back, but I figured it would be easy to deprecate this now while we're already making breaking changes in the FluentUIFramework, otherwise we may never get to it.